src/raven.js

```diff
@@ -416,11 +416,14 @@ Raven.prototype = {
       return this.captureMessage(
         ex,
         objectMerge(
+          objectMerge(
+            {
+              stacktrace: true // if we fall back to captureMessage, default to attempting a new trace
+            },
+            options),
           {
-            trimHeadFrames: 1,
-            stacktrace: true // if we fall back to captureMessage, default to attempting a new trace
-          },
-          options
+            trimHeadFrames: ((options && options.trimHeadFrames) || 0) + 1,
```

Comment:
Can we set the default for `options` at the top of the method while also making a copy?

```
options = options ? Object.merge({}, options) : {}
```

and then just leave `(options.trimHeadFrames || 0) + 1`?

We also don't need a double `objectMerge`, as it can be done in a single call when we do so:

```js
objectMerge(options, {
  stacktrace: true,
  trimHeadFrames: (options.trimHeadFrames || 0) + 1
})
```

Although I'm not sure if we cannot just reverse it, even though it's reference and write

```
objectMerge(options, {
  stacktrace: true,
  trimHeadFrames: (options.trimHeadFrames || 0) + 1
})
```

without making a copy.

Reply:
Sounds like a good idea!